src/plugin/index.js

```diff
@@ -19,7 +21,7 @@ function Jarvis(options) {
   };
 }
 
-Jarvis.prototype.apply = function(compiler) {
+Jarvis.prototype.apply = function (compiler) {
```

Comment:
Sorry, forgot to disable prettier before making the changes so it added this redundant space here :D

Reply:
Please feel free to use prettier! Just with the native prettier configs would be good!